src/components/Dialog.tsx

```diff
@@ -202,9 +203,21 @@ const fadeAnimationCss = css`
   }
 `
 
-const AnimationWrapper = styled.div<{ animationType: DialogAnimationType }>`
+const getAnimation = (animationType?: DialogAnimationType) => {
+  switch (animationType) {
+    case DialogAnimationType.NONE:
+      return css``
```

Comment:
can we return a build-time constant here that won't re-run the template string function call when `getAnimation` runs?

Reply:
(just shift the empty css`` call up out of the fn)